### DISCUSSION TITLE
Guess the song 🎸

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1165] [Galaxy Trivia] [answered]
```
E|------------------------------------------------|
B|--8--8--8--8--7--7--7--7-----------5----5--5----|
G|--9--9--9--9--7--7--7--7--5--5--5--5----5--5----|
D|--------------------------7--7--7---------------|      (Play 4X)
A|------------------------------------------------|
E|------------------------------------------------|
```

**Bonus:** Name the songwriter

**Comment by ParanoidUser** [comment]
@Better-zyh, any idea what song that might be? 🎤🎷🎸

**Comment by ParanoidUser** [comment]
@Sajansharma0017, want give it a try?

**Reply by Sajansharma0017** [reply]
@ParanoidUser  can I get a  Hint?

**Reply by Sajansharma0017** [reply]
Is this somewhat related to angel numbers??

**Reply by ParanoidUser** [reply]
Check out the comment below 😉.

**Comment by thuongtruong1009** [comment]
The song you are referring to is "Smoke on the Water" by Deep Purple. The songwriter is Ritchie Blackmore, Ian Gillan, Roger Glover, Jon Lord, and Ian Paice. This iconic riff is one of the most recognizable guitar riffs in rock music history.

**Reply by ParanoidUser** [reply]
Nice try!

**Comment by ParanoidUser** [comment]
**Hint:** Solve this puzzle 🧩 to reveal the band.

```
I'm not from here, that's plain to see,
In a different land, I used to be.
I may not know your customs and ways,
But I'm a visitor for just a few days.
```

**Comment by peterbaccaro** [comment, marked as answer]
The song is **Urgent** by the British-American rock band **Foreigner** ([tab link](https://tabs.ultimate-guitar.com/tab/foreigner/urgent-tabs-1709522)).
The song was written by **Mick Jones** who was the guitarist and key songwriter for the band.

**Reply by ParanoidUser** [reply]
Thank you, Peter! 👏👏
